front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -277,8 +295,15 @@ export default {
       maxPage: 0,
       imageList: [],
       splitLength: 1000,
+      allImageLength: 1000,
+      correctImageLength: 1000,
```

Comment:
The initial accuracy may be seen by the user before the frontend receives a response from `this.initClassifiedImageList()`. 

The initial accuracy will be 1 if you initialize all these numbers to be 1000. IMO it might be better to initialize  `correctImageLength` to `-1` so that the accuracy is negative initially, which will let users know it is abnormal and they probably need to wait. Otherwise users may think `1` is really the accuracy of their model.